src/components/Search/SearchFeedback.scss

```diff
@@ -1,4 +1,4 @@
-.chr-c-search-feedback {
+.chr-c-search-feedback.pf-v5-c-menu__group {
```

Comment:
My selector wasn't specific enough causing some weirdness when interacting with the search results. This resolves the issue.